src/index.js

```diff
@@ -1,2 +1,73 @@
 import './style.css';
-alert("hello world");
\ No newline at end of file
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const toDoArray = [];
+const listContainer = document.querySelector('.list-container');
+const input = document.querySelector('#add-todo-input');
+const form = document.querySelector('#add-todo');
+
+class ToDo {
+  constructor(description, completed = false, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+
+  static addTodo(todo, arr) {
+    arr.push(todo);
+    todo.index = arr.indexOf(todo);
+  }
```

Comment:
- [ ] The code is nice, but as its just the beginning you have to add an array of object  that has an index, description, and complete as true/ false instead of pushing it. 

![gfgfg](https://user-images.githubusercontent.com/61562626/134380012-9f5b11c1-181f-476c-9649-e9d99f502772.jpeg)